redis/deprecated/redisbp/monitored_client.go

```diff
@@ -2,102 +2,40 @@ package redisbp
 
 import (
 	"context"
-	"io"
 	"time"
 
-	"github.com/go-redis/redis/v7"
+	"github.com/go-redis/redis/v8"
 	"github.com/reddit/baseplate.go/metricsbp"
 )
 
-// MonitoredCmdable wraps the redis.Cmdable interface and adds additional methods
-// to support integrating with Baseplate.go SpanHooks.
-type MonitoredCmdable interface {
-	// Close should generally not be called directly on a MonitoredCmdable, since
-	// they are meant to be shared and long lived.  It will be called by
-	// MonitoredCmdableFactory.Close which should be called when a server is shut
-	// down.
-	io.Closer
-	redis.Cmdable
-
+// PooledCmdable is an interface with PoolStats that reports pool related metrics
+type PooledCmdable interface {
```

Comment:
since this does not embed `redis.Cmdable`, maybe we shouldn't add `-Cmdable` to its name?

Reply:
How'bout `Poolable` ?

Reply:
Or maybe `MonitoredPool`?

Reply:
In go convention, since this is an interface with only one function, it's supposed to be called `PoolStater` (I removed the `-s`).

Reply:
Or PoolStatser since Stater is an ancient Greek gold or silver coin :smile:

Reply:
ha! let's agree on `PoolStatser` ?